package.json

```diff
@@ -35,23 +35,26 @@
     "url": "https://github.com/rangle/angular2-redux-starter/issues"
   },
   "dependencies": {
-    "@types/jasmine": "^2.2.33",
-    "@types/node": "^6.0.38",
     "http-server": "^0.9.0"
   },
   "devDependencies": {
-    "@angular/common": "2.0.0-rc.6",
-    "@angular/compiler": "2.0.0-rc.6",
-    "@angular/core": "2.0.0-rc.6",
-    "@angular/http": "2.0.0-rc.6",
-    "@angular/platform-browser": "2.0.0-rc.6",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.6",
+    "@angular/common": "2.0.0-rc.7",
+    "@angular/compiler": "2.0.0-rc.7",
+    "@angular/compiler-cli": "^0.6.1",
+    "@angular/core": "2.0.0-rc.7",
+    "@angular/http": "2.0.0-rc.7",
+    "@angular/platform-browser": "2.0.0-rc.7",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.7",
+    "@angular/platform-server": "2.0.0-rc.7",
     "@angular/router": "^3.0.0-rc.2",
+    "@types/jasmine": "^2.2.33",
+    "@types/node": "^6.0.38",
+    "@types/core-js": "^0.9.32",
     "autoprefixer": "^6.3.3",
     "awesome-typescript-loader": "^2.2.1",
     "babel-core": "^6.9.1",
```

Comment:
If we're getting typescript to emit es5 code, then we can get rid of babel-core, babel-loader, babel-polyfill, and babel-preset-es2015.